### DISCUSSION TITLE
Vacuum Integration

### DISCUSSION THREAD

**Discussion by BakaNode** [SecKatie/ha-wyzeapi#176] [Ideas]
My apologies if this is redundant but I could not seem to find a clear answer on the other two open threads.

Is the vacuum integration currently on your roadmap to support?

I’ve been watching the progress on your project for about a month now and I am very impressed and willing to start supporting financially once I see some kind of response on the vacuum.

Also, is this ever going to be added to the official integrations for Home Assistant so I don’t have to dirty my install with HACS?

P.S. Sent you a coffee in advance to show my support.

**Comment by BakaNode** [comment]
It’s been 19 days and not a peep on this thread. That’s disheartening and doesn’t bode well for future support.

**Reply by SecKatie** [reply]
Sorry for the late reply. This is not on the roadmap.

**Reply by SecKatie** [reply]
It will likely never be an official integration either. I prefer having control of the distribution of this integration.
